x-pack/plugins/snapshot_restore/__jest__/client_integration/home.test.ts

```diff
@@ -409,9 +408,9 @@ describe('<SnapshotRestoreHome />', () => {
 
         await act(async () => {
           testBed.actions.selectTab('snapshots');
-          await delay(100);
```

Comment:
`delay()` is not necessary and `component.update()` should be called outside of `act()`